src/services/evm/crawl_evm_transaction.service.ts

```diff
@@ -36,13 +40,19 @@ export default class CrawlEvmTransactionService extends BullableService {
     }
 
     const blocks = await EVMBlock.query()
-      .select('height', 'transactions', 'tx_count')
+      .select('height', 'transactions', 'tx_count', 'date')
       .where('height', '>', startBlock)
       .where('height', '<=', endBlock)
       .orderBy('height', 'asc');
 
-    const offchainTxs: FormattedTransaction[] = blocks
-      .map((block) => block.transactions)
+    const offchainTxs: CustomFormattedTransaction[] = blocks
+      .map((block) => {
+        block.transactions.forEach((tx: any) => {
+          // eslint-disable-next-line no-param-reassign
+          tx.date = block.date;
```

Comment:
ủa block.time chứ nhỉ?

Reply:
.date anh ạ

Reply:
ơ mà date thật, chắc đặt tên nhầm rồi
em cho lại thành block.timestamp luôn nhé

Reply:
fixed in block and tx